nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/computation/AbstractComputation.java

```diff
@@ -65,4 +84,32 @@ public void processTimer(ComputationContext context, String key, long timestamp)
     public ComputationMetadata metadata() {
         return metadata;
     }
+
+    public ComputationPolicy getPolicy() {
+        return policy;
+    }
+
+    public void setPolicy(ComputationPolicy policy) {
```

Comment:
Shouldn't be protected as this method is used only in tests + shouldn't be something editable no ?

Reply:
yes, I will set it to protected